src/pages/codingClubs/CodingClubs.tsx

```diff
@@ -0,0 +1,99 @@
+      <page.Banner
+        imageProps={{ alt: "codeClubHero", src: CodeClubHeroImage }}
+        header="Coding clubs"
+        subheader="A FREE set of slides and guides to run your own coding clubs"
+      />
+      <page.Section>
+        <ClubAim />
+      </page.Section>
+      <page.Section boxProps={{ bgcolor: theme.palette.info.main }}>
+        <Introduction
+          header="Primary coding club"
+          img={{ alt: "aboutUs", src: AboutUsImage }}
+        >
+          <Typography>
+            Download your FREE coding club pack for students aged 7-11. This
+            pack introduces students to the first principles of Python at a
+            faster pace than the regular lesson plans. It is aimed at students
+            already interested in learning coding and can be used in clubs, at
+            home or in school, on or offline.
+          </Typography>
+          <Typography>
+            View the resources{" "}
+            <Link
+              to="https://code-for-life.gitbook.io/teaching-resources/v/code-club-resources-primary"
+              target="_blank"
+            >
+              online here
+            </Link>
+            .
+          </Typography>
+          {/*TODO: Link to GTM for analytics*/}
+          <LinkButton
+            sx={{ marginTop: "auto" }}
+            to="https://storage.googleapis.com/codeforlife-assets/club_packs/PrimaryCodingClub.zip"
+            target="_blank"
+            endIcon={<DownloadIcon />}
+          >
+            Download the Primary coding club pack
+          </LinkButton>
+        </Introduction>
+      </page.Section>
+      <page.Section>
+        <Introduction
+          header="Python coding club"
+          img={{ alt: "pythonCodingClub", src: PythonClubImage }}
+          direction="row-reverse"
+        >
+          <Typography>
+            Download your FREE coding club pack for students aged 12 and above.
+            This pack is a fast paced introduction to Python. It is aimed at
+            students already interested in learning coding, individuals looking
+            to learn and run their own club, or adults wanting to try coding
+            out. It is designed to be used in face-to-face or online clubs.
+          </Typography>
+          <Typography>
+            View the resources{" "}
+            <Link
+              to="https://code-for-life.gitbook.io/teaching-resources/v/rapid-introduction-to-python-code-club"
+              target="_blank"
+            >
+              online here
+            </Link>
+            .
+          </Typography>
+          {/*TODO: Link to GTM for analytics*/}
+          <LinkButton
+            sx={{ marginTop: "auto" }}
+            to="https://storage.googleapis.com/codeforlife-assets/club_packs/PythonCodingClub.zip"
+            target="_blank"
+            endIcon={<DownloadIcon />}
+          >
+            Download the Python coding club pack
+          </LinkButton>
+        </Introduction>
```

Comment:
Make this a page component (like <ClubAim />) called <Python />. This will help readability - no single large components but components made of smaller components. Generally a good idea to make page sections a component. See home page as an example.